src/test/java/com/exasol/adapter/document/DocumentAdapterPropertiesTest.java

```diff
@@ -0,0 +1,93 @@
+package com.exasol.adapter.document;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import com.exasol.adapter.AdapterProperties;
+
+class DocumentAdapterPropertiesTest {
+    private static final String MAPPING_PATH = "/bfsdefault/default/mapping.json";
+    private static final String MAPPING_KEY = "MAPPING";
+    private static final String MAX_PARALLEL_UDFS_KEY = "MAX_PARALLEL_UDFS";
+
+    @Test
+    void testGetMappingDefinition() {
+        final DocumentAdapterProperties properties = new DocumentAdapterProperties(
+                new AdapterProperties(Map.of("MAPPING", MAPPING_PATH)));
+        assertThat(properties.getMappingDefinition(), equalTo(MAPPING_PATH));
+    }
+
+    @Test
+    void testGetMappingDefinitionRemovesBucketsPrefix() {
+        final DocumentAdapterProperties properties = new DocumentAdapterProperties(
+                new AdapterProperties(Map.of(MAPPING_KEY, "/buckets" + MAPPING_PATH)));
```

Comment:
I wonder what would happen if the mapping path is something like `/bucketswhatever`.

We should add a test for it.